packages/frontend-2/components/viewer/anchored-point/Thread.vue

```diff
@@ -151,6 +151,21 @@
               :model-value="modelValue"
               @submit="onNewReply"
             />
+            <div
+              v-if="!canReply"
+              class="p-3 flex flex-col items-center justify-center bg-foundation-2"
+            >
+              <p class="text-sm mb-2 font-bold">Join the conversation</p>
+              <FormButton size="sm" @click="showLoginDialog = true">
+                Sign in to Speckle
+              </FormButton>
+              <AuthLoginPanel
+                v-model:open="showLoginDialog"
+                dialog-mode
+                max-width="sm"
+                subtitle="Create a free account to keep using Speckle!"
+              />
+            </div>
```

Comment:
I like your "Join the conversation" copy but I suggest we just have a blue "Reply" button and clicking that opens the sign in dialog.

<img width="482" alt="Screenshot 2024-02-02 at 23 11 33" src="https://github.com/specklesystems/speckle-server/assets/2694102/ea452d6b-6e84-4692-b6f9-a962041edffa">

Reply:
<img width="440" alt="image" src="https://github.com/specklesystems/speckle-server/assets/139135120/dc23a8bf-4e8f-4b6e-83c3-702ac57a1d10">

Reply:
@andrewwallacespeckle Can you update it to what's in my image instead? No "Join the conversation" text and just a big Reply button (full-width button and a bit taller than what's in your image). Now that I'm thinking about it we could then use the "Join the conversation" copy as the subtitle in the auth modal.

Reply:
Ah sorry, I misread your comment. Will update now

Reply:
<img width="425" alt="image" src="https://github.com/specklesystems/speckle-server/assets/139135120/b10f48b9-63d6-4ef7-bfb5-1273a2c4ab73">

Reply:
I've made this change and pushed. There is a bit more work in modifying the Dialog to include "join the conversation", but it can be done in about 10 mins. Please confirm if you want me to do this